packages/core/components/StatusMessage/index.tsx

```diff
@@ -23,6 +23,8 @@ const statusToTypeMap = {
     [ProcessStatus.SUCCEEDED]: MessageBarType.success,
     [ProcessStatus.FAILED]: MessageBarType.error,
     [ProcessStatus.NOT_SET]: MessageBarType.info,
+    [ProcessStatus.WARNING]: MessageBarType.severeWarning,
+    [ProcessStatus.ERROR]: MessageBarType.severeWarning,
```

Comment:
good question, I went back and forth on this.
Since this component used to only be used for processes in progress, "Failed" was the existing class (w old icon) used for when that process failed to complete. I added error with the new icon from figma in case we wanted to use this modal for general system errors. Could definitely be combined, I'd just need to check with Lyndsay which of the icons we prefer to keep.

Here's failed, error and warning for comparison:
<img width="767" alt="image" src="https://github.com/user-attachments/assets/f570739a-ce09-4662-a372-453325cf6a02">

Reply:
I don't particularly catch which we keep, but I think we should choose betwee ERROR or FAILURE

Reply:
I'm going with error then since it's more generic sounding